### PR TITLE
Fix top position issue when window is scrolled up and parents has fix position.

### PR DESCRIPTION
**Pull request by jianwu**
Fix top position issue when window is scrolled up and the input or textarea's parents have fixed position setting. Which causes problem when input/textarea is inside  Bootstrap modal dialog, when the main page has been scrolled up. The drop list position will be far blow the expected position.

This is caused by the wrong offset calculation. jQuery offset is related to document instead of view port. And in fixed position the top is related to view port. 

This fixes following issues: 

Problem with placement top #125
Calculate positioning relative to the appendTo element #206

### CODE REVIEW

File: src/dropdown.js [hunk 1]
```diff
@@ -143,10 +141,13 @@
         if($(this).css('position') === 'absolute') // The element has absolute positioning, so it's all OK
           return false;
         if($(this).css('position') === 'fixed') {
+          pos.top -= $(window).scrollTop();
+          pos.left -= $(window).scrollLeft();
```
Comment (yuku-t) [inline src/dropdown.js]: There is `$window = $(window);`. please use it instead :)

**Comment by yuku-t** [comment]
Sorry for my late reply. It looks almost all good to me. Please check my comment. When you fix it, I'll merge this pull request.

**Comment by jianwu** [comment]
I have changed accordingly. Thanks the project, it's very well implemented.

**Comment by yuku-t** [comment]
thx

**Pull request merged**